charts/loki-canary/values.yaml

```diff
@@ -21,6 +21,9 @@ priorityClassName: null
 # -- Common annotations for all pods
 podAnnotations: {}
 
+# -- Common labels for all pods
+podLabels: {}
```

Comment:
WDYT of making this `selectorLabels` or `podSelectorLabels` to match the convention in the helper function `loki-canary.selectorLabels` well call right before including this?

Reply:
ahh, so you want to differentiate these from `selectorLabels` as in they are _only_ meant for the pod. I agree with that argument